src/index.ts

```diff
@@ -1 +1,113 @@
+/**
```

Comment:
this is the readme. I created a nest / typeorm app and added admin bro there. Please check it out if it is the correct setup, because this will appear in the adminbro.com/docs